torch/utils/cpp_extension.py

```diff
@@ -1724,3 +1740,10 @@ def _is_cuda_file(path):
     if IS_HIP_EXTENSION:
         valid_ext.append('.hip')
     return os.path.splitext(path)[1] in valid_ext
+
+def _to_absolute_path(paths):
+    if not paths:
+        return
```

Comment:
This special case is totally unnecessary since the loop will quit out if the paths array is empty

Reply:
This is to prevent calling len() on None which will result in a crash.